content/go/CDN_services.go.md

```diff
@@ -0,0 +1,568 @@
+}
+```
+
+### CreateDomainMapping
+
+```
+package main
+
+import (
+"encoding/json"
+"fmt"
+"github.com/softlayer/softlayer-go/datatypes"
+"github.com/softlayer/softlayer-go/services"
+"github.com/softlayer/softlayer-go/session"
+"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	// SoftLayer API username and key
+	username := "set - me"
+	apikey := "set - me"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+
+	templateCdn:= datatypes.Container_Network_CdnMarketplace_Configuration_Input{
+		Header: sl.String("www.techsupport.com"),
+		Cname: sl.String("cdnakaogtechSpport.cdn.appdomain.cloud"),
+		Domain: sl.String("www.testGotest.com"),
+		Path: sl.String("/example"),
+		Protocol: sl.String("HTTPS"),
+		Origin: sl.String("10.10.15.2"),
+		OriginType: sl.String("HOST_SERVER"),
+		VendorName: sl.String("akamai"),
+		HttpsPort: sl.Int(80),
+	}
+
+	receipt, err := service.CreateDomainMapping(&templateCdn)
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+		return
+	}
+
+	// Following helps to print the result in json format.
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+		return
+	}
+	fmt.Println(string(jsonFormat1))
+}
+```
+
+### DeleteDomainMapping
+
+```
+package main
+
+
+import (
+"encoding/json"
+"fmt"
+"github.com/softlayer/softlayer-go/services"
+"github.com/softlayer/softlayer-go/session"
+"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	// SoftLayer API username and key
+	username := "set - me"
+	apikey := "set - me"
+
+	// Create SoftLayer API session
+ 	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+    // Input the unique Id 
+	uniqueId := sl.String("123456789")
+
+	receipt, err := service.DeleteDomainMapping(uniqueId)
+
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+		return
+	}
+
+	// Following helps to print the result in json format.
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+		return
+	}
+	fmt.Println(string(jsonFormat1))
+}
+```
+
+### VerifyDomainMapping
+
+```
+package main
+
+import (
+"encoding/json"
+"fmt"
+"github.com/softlayer/softlayer-go/services"
+"github.com/softlayer/softlayer-go/session"
+"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	// SoftLayer API username and key
+	username := "sl307608-dcabero"
+	apikey := "8f639dfb380af0667ee4949cdf2dbc9eb8e20b2d777e92aba7312f24a9524988"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	service := services.GetNetworkCdnMarketplaceConfigurationMappingService(sess)
+	uniqueId := sl.Int(723231201263657)
+
+	receipt, err := service.VerifyDomainMapping(uniqueId)
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
```

Comment:
Make this error more generic, since this isn't placeing an order.

fmt.Printf("\nAPI Error:\n - %s\n", err)